app/views/layouts/_global_header.html.haml

```diff
@@ -26,7 +26,7 @@
 
     :javascript
       ManageIQ.browser = "#{browser_info("name")}";
```

Comment:
Is this one really safe? I can see why Hakiri wouldn't have a problem with it, but without actually finding the definition of `browser_info`, it sure sounds like it could have the same problem.